packages/fxa-auth-server/lib/routes/subscriptions/paypal.ts

```diff
@@ -110,8 +108,7 @@ export class PayPalHandler extends StripeWebhookHandler {
 
       // Creating a subscription with automatic_tax enabled requires a customer with an address
       // that is in a recognized location with an active tax registration.
-      const taxSubscription =
-        this.automaticTax && customer.tax?.automatic_tax === 'supported';
+      const taxSubscription = customer.tax?.automatic_tax === 'supported';
```

Comment:
Looks like we have this pattern 4&times; in these files (2&times; in stripe.ts and 2&times; in paypal.ts). Not sure if that's a candidate for moving it into some helper function.
Also interesting is that one instance is slightly different logic (which might not be a bug, but was just something curious to my suspicious eyes):
```js
    // has an additional check for `!customer`.
    const isCustomerTaxed =
      !customer || customer.tax?.automatic_tax === 'supported';
```

Reply:
I noticed this too, though the comment mentions that this requires a customer [with an address that is in a recognized location with an active tax registration].

@julianpoy, can you confirm how `taxSubscription` should now be defined here?

Reply:
The extra check for customer in the instance @pdehaan is referencing is needed since previewInvoice does not require a current customer. Credential validation is optional on that handler, so the customer can be null in that instance.